
Fix for gcc >= 3.4.x - the usual suspects.

  - Rene Rebe <rene@exactcode.de>

--- ./app/keys.c.vanilla	2004-11-05 12:53:13.496797548 +0100
+++ ./app/keys.c	2004-11-05 12:53:26.182611773 +0100
@@ -765,6 +765,7 @@
 	gnome_warning_dialog(_("The keyboard configuration file is defective.\nPlease use the Keyboard Configuration dialog."));
 	return 0;
       break2:
+	;
     }
     r = 1;
 
